workshops/models.py

```diff
@@ -726,6 +755,10 @@ class Event(AssignmentMixin, models.Model):
         on_delete=models.PROTECT,
         help_text='Organization responsible for administrative work.'
     )
+    sponsors = models.ManyToManyField(
+        Host, related_name='sponsor', blank=True,
```

Comment:
Am I reading this correctly? `Event.sponsors → Hosts` but `Host.sponsor → Event`?